app/Console/Commands/ImportMRES.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use Illuminate\Console\Command;
+
+class ImportMRES extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'import:mres {input} {output} {--networks=}';
+
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'One-off script to import MRES groups.  This takes a CSV of the groups in MRES format and outputs a CSV in our standard import format.  You can convert from the ODS file to CSV; make sure you select "Western Europe (ISO08859-15/EURO)" as the Character set.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $input = $this->argument('input');
+        $output = $this->argument('output');
+        $networks = $this->option('networks');
```

Comment:
What's the format of this - comma separated list of networks ids?

Reply:
Yes.